src/main/java/org/sasanlabs/beans/LevelResponseBean.java

```diff
@@ -3,16 +3,24 @@
 import com.fasterxml.jackson.annotation.JsonProperty;
 import java.util.ArrayList;
 import java.util.List;
+import java.util.function.ToIntFunction;
 import org.sasanlabs.internal.utility.LevelConstants;
+import org.sasanlabs.internal.utility.SecureConstants;
+import org.sasanlabs.internal.utility.VariantConstants;
 import org.sasanlabs.internal.utility.annotations.RequestParameterLocation;
 import org.springframework.web.bind.annotation.RequestMethod;
 
-/** @author KSASAN preetkaran20@gmail.com */
+/**
```

Comment:
can you please run `gradle spotlessApply` ?

Reply:
Resolved in 422b8d3